### DISCUSSION TITLE
Best Vision Paper Insight 1주차~6주차

### DISCUSSION THREAD

**Discussion by geonhak904** [Pseudo-Lab/Best_Vision_Paper#2] [Ideas]
본 Discussion 장소를 통해 회차별 논문에 대한 자신만의 Insight를 공유하며 왜 이 논문이 어떤 의미를 가지는지 한 번 고민해보는 시간을 가져봅니다.

(Optional) 질문 (최소 하나 이상의 질문 권장)
(Optional) 발표에 포함시켜줬으면 하는 내용 
(Optional) ... etc

**Comment by geonhak904** [comment]
[ 발표 예정일 : 2025.03.11 (화) ] 
Generative Image Dynamics (CVPR 2024, Best Paper Award)

[Project](https://generative-dynamics.github.io/)
[Arxiv](https://arxiv.org/abs/2309.07906)

발표자 : 송건학

**Reply by jiny419** [reply]
### Best paper 이유
single image로부터 spectral volume (frequency-domain)을 활용하여 long-range pixel trajectory을 학습하고 이를 기반으로 모션을 생성하는 접근법을 제안함. 또한, diffusion 모델을 활용하여 분포를 학습하고 자연스러운 motion을 생성할 수 있도록 설계함. image-to-video, seamless looping video, interactive dynamic image를 포함한 다양한 비디오 생성분야에 적용될 수 있음. 

###  Insight
- adaptive norm 방법에 square root form 적용. ablation study 결과에서도 adaptive norm방법이 큰 영향을 끼침.
- per-pixel weight로 flow magnitude 사용.

### limitation 
- 세세한 떨림이 필요한 모션 생성에 맞춰줘있기 때문에 주로 확대된 물체 중심의 영상에 적합함.

**Reply by kteacher94** [reply]
**Best Paper인 이유**

기존의 영상기반으로 RGB 픽셀의 Optical Flow를 포함하여 motino과 관련된 feature를 학습한 diffusion 모델과 달리 푸리에(주파수) 도메인에서 피사체의 흔들림 feature를 학습 및 모델링하여 보다 효과적으로 단일 이미지 입력만으로 피사체가 흔들리는 모션을 추론 및 예측할 수 있는 점이 색다르다.

**Reply by hoon99** [reply]
기존 연구들은 정적 이미지에서 자연스러운 움직임을 예측하는 데 한계가 있었는데, 이 연구에서 Fourier 도메인의 Spectrum Volume 표현과 주파수 대역 조정용 Diffusion model을 활용해서 정교한 패턴을 만들어냄. 다른 방법들은 짧은 시간 동안의 모션만 예측했던 반면, 이 논문은 장시간에 걸친 연속적이고 자연스러운 모션을 생성할 수 있다는 차별점이 있음

**Reply by cuchoco** [reply]
Spectral volume이라는 feature를 주파수 도메인에서 예측하는 모델을 통해 보다 자연스럽고 장기적인 움직임을 효과적으로 학습하였다. 또한, 단일 이미지에서도 사용자의 interaction에 따라 Modal Analysis와 Explicit Euler Method를 적용해 객체가 물리적으로 자연스럽게 반응하도록 구현하였다는게 큰 기여로 보인다.

**Comment by hoon99** [comment]
[ 발표 예정일 : 2025.03.18 (화) ]

**Is ImageNet worth 1 video? Learning strong image encoders from 1 long unlabelled video** (ICLR 2024, Honorable Mention)

[Arxiv](https://arxiv.org/abs/2310.08584)
[Github](https://github.com/shashankvkt/DoRA_ICLR24)

발표자 : 고재훈

**Reply by geonhak904** [reply]
## Insight
**새로운 Dataset: Walking Tours**
- 단일 Walking Tours 비디오만으로도 이미지 및 비디오 downstream task에서 ImageNet과 경쟁할 수 있는 강력한 성능을 달성함.
- 해당 데이터 세트는 고해상도, 긴 시간의 연속적 1인칭 영상으로 구성되어 있으며, 자연스러운 장면 전환과 풍부한 객체 정보 덕분에 SSL에 최적화됨.

**DORA: 새로운 Self-Supervised Learning(SSL) 방법**
- DINO는 영상 내 객체를 tracking하면서 학습을 진행함. Transformer의 cross-attn과 Sinkhorn-Knopp을 활용해 객체 간 일관성을 유지하며, 학습 성능을 크게 향상시킴.

## Honorable Mention 사유
- Novelty (새로움): 데이터 규모 중심의 접근에서 벗어나 단, 1개의 unlabeled video로 ImageNet 수준의 학습이 가능하다 점을 통한 참신성.
- Efficiency (효율성): 본 방법을 통해 제공되는 새로운 데이터셋과 간결한 모델 구성으로 학습 자원 절약 및 확장성 확보, 실질적 활용 가능.

## 한계점
- 제한된 범위: 단일 영상 기반 학습은 분명 의미있는 시도이지만, 당시 대규모 데이터를 활용한 규모의 확장성 측면에서 분포적 확장성이 훨씬 더 효과적이고 효율적이며 더 정교한 결과를 만들 수 있는 측면이 커 실질적 점수에는 어려움이 있었을 것으로 판단됨

**Reply by jiny419** [reply]
### Honorable mention
- 대규모 객체 중심의 인터넷 비디오에 의존하는 대신, 이 연구는 egocentric Walking tours 비디오 데이터셋을 소개함. 이러한 비디오는 실제 인간의 시각적 경험을 더욱 잘 반영하며, 인위적인 편집 없이 연속적이고 의미적으로 풍부한 프레임을 제공함.
- 저자들은 인간의 인지 발달에서 영감을 받은 DORA라는 방법을 제안하며, 이는 프레임 간 객체를 추적하여 인식 능력을 향상시킴. 이 전략은 기존의 데이터 증강 기반 접근 방식과 차별화되며 학습 효율성을 높임.

### ICLR official ([Link](https://blog.iclr.cc/2024/05/06/iclr-2024-outstanding-paper-awards/))
- The paper proposes a novel path to self-supervised image pre-training, by learning from continuous videos. The paper contributes both new types of data and a method to learn from novel data.

**Reply by cuchoco** [reply]
이 연구는 하나의 긴 비디오가 대규모 이미지 데이터셋(ImageNet)만큼의 visual representation 학습 가능성을 보여줌. 
DoRA라는 프레임워크로 비디오 데이터를 효율적으로 학습해 이미지넷과 같이 
많은 레이블 없이도 효과적으로 이미지 인코더를 학습 할 수 있다는 점에서 가치가 있음.

\+ Sinkhorn-Knopp 알고리즘에 대해 조금 더 설명을 해주시면.. 감사하겠습니다.

**Reply by kteacher94** [reply]
새로운 Waling Tours 데이터셋을 활용한 기존 방법론의 대용량 이미지 데이터셋의 학습 없이도 상대적으로 적은 데이터셋에서 이미지 인코더를 효과적으로 학습할 수 있는 방법론을 제시하였고, 트랜스포머의 크로스 어텐션을 통해 객체를 발견 및 추적하는 Dora 모델을 도입한 점이 새롭다.

**Comment by kteacher94** [comment]
pixelSplat: 3D Gaussian Splats from Image Pairs for Scalable Generalizable 3D Reconstruction (Honorable Mention CVPR2024)

[Arxiv](https://arxiv.org/abs/2312.12337)

[Github](https://davidcharatan.com/pixelsplat/)

**Reply by geonhak904** [reply]
## Insight
**1. Scale Ambiguity 문제 해소법**
Multi-view epipolar transformer를 설계하여 이미지 간 픽셀 대응 관계를 찾고, 장면의 스케일을 추론. 이를 통해 정확한 깊이 예측 가능.

**2. Gaussian Primitives Parameter 예측**
3D 공간에 대한 밀도 확률 분포를 예측하고, 이를 기반으로 Gaussian means를 샘플링.

**3. 효율적인 rendering**
3D Gaussian Splatting을 활용하여 기존 volume rendering 방식보다 빠르고 효율적인 실시간 렌더링 제공. 이때 sampling 연산을 reparameterization trick을 통해 미분가능하게 만들어 최적화 진행

## Honorable Mention 달성 요인
**Novelty :** 
3D Gaussian Splatting을 sparse novel view synthesis에서 3D radiance field를 예측하는 새로운 feed-forward model을 제안함으로써, 기존 대비 차별화된  일반화 기법을 제시
**Efficiency :** 
효율적인 렌더링과 낮은 메모리 사용량으로 기존 방법 대비 훈련 및 추론 비용을 감소(기존 대비 650배 빠른 렌더링).
**Performance :** 
benchmark dataset에서 SOTA급 성능과 지표 결과.

## 한계점
- 제한된 범위 및 검증 : 2장의 sparse image를 통한 3D radiance field를 재구성이 가능함을 보이나 occlusion 혹은 데이터가 부족한 영역에서의 모델링에 있어 아쉬운 부분이 존재함. 또한, 일반화에 대한 실질적 검증 부분에 있어 부정확한 정보 처리에 대한 활용성의 아쉬움 존재

## 후속 논문
1) [MVSplat360](https://zezeaaa.github.io/projects/MVPGS/) - ECCV 2024
2) [Flash3D](https://www.robots.ox.ac.uk/~vgg/research/flash3d/): Feed-Forward Generalisable 3D Scene Reconstruction from a Single Image

**Reply by jiny419** [reply]
### Honorable mention
- differentiable rendernig 기법이 갖는 높은 메모리 및 계산 비용 문제를 해결하기 위해 3D Gaussian primitive 를 활용하여 빠르고 메모리 효율적인 렌더링을 가능하게 함.
- 기존 연구들은 non-differentiable 연산이나 heuristic을 활용해야 했던 반면, pixelSplat은 3D Gaussian splatting을 완전히 미분 가능하도록 통합하여 학습 및 최적화 수행 함.

### [HiSplat (ICLR 2025)](https://github.com/Open3DVLab/HiSplat)
![Image](https://github.com/user-attachments/assets/b0151e89-a3dc-4b19-a255-5ee1416f1901)

**Reply by cuchoco** [reply]
1. 3D Gaussian 기반의 표현 학습
기존 방식은 많은 이미지와 계산을 필요로 하지만, pixelSplat은 단 두 장의 이미지에서 3D Gaussian primitive를 예측해 novel view를 생성.
2. Scale Ambiguity 해결 
다수의 데이터셋에서는 카메라 포즈가 임의의 스케일로 복원되는데 이를 해결하기 위해 epipolar attention 기반의 Transformer를 사용하여 상대적 스케일을 예측.
3. local minima 해결
Gaussian 위치를 직접 최적화하면 local minima에 빠질 수 있음, 확률적 sampling과 reparameterization trick으로 differentiable하게 위치를 예측하여 해결.

Honorable mention의 이유: 
기존 방식 대비 렌더링 속도를 대폭 개선했으며, 
기존 3D Gaussian splatting 기법은 수작업 최적화가 필요했지만, 이 논문은 이를 end-to-end로 해결 시도.

**Reply by hoon99** [reply]
1. dense 확률 분포를 예측하고 sampling + reparameterization trick을 도입함으로써, 3D Gaussian 위치를 완전히 end-to-end 미분 가능한 형태로 예측함
2. Pixel-aligned Gaussian이라는 새로운 primitive 정의를 통해 2D 이미지의 각 픽셀에서 직접적으로 대응되는 3D 공간상의 위치와 특성을 예측

**Comment by kmscopragit** [comment]
Rich Human Feedback for Text-to-Image Generation (CVPR2024, Best Paper Award)

[Arxiv](https://arxiv.org/abs/2312.10240)

[Github](https://github.com/google-research/google-research/tree/master/richhf_18k)

**Comment by cuchoco** [comment]
Protein Discovery with Discrete Walk-Jump Sampling (ICLR 2024, Outstanding Papers)

[Arxiv](https://arxiv.org/abs/2306.12360)

**Reply by geonhak904** [reply]
단백질 디자인은 거대하고 이산 공간에서 유용한 단백질을 찾는 문제로 제한된 데이터셋 내에서 sample 효율성이 중요. 기존 방법들(EB model, autoregressive model etc)에서는 효율적이지 않거나 오류가 높은 문제들이 있었음.

# Insight

**1. Smoothed Discrete Sampling (SDS)** : 기존 EBM과 Score-Based Diffusion Model의 결합을 통해 smoothed data manifold에서 효율적 샘플링 제안. 또한, Single Noise Level Training을 통한 훈련 안정성 향상

**2. Discrete Walk-Jump Sampling (dWJS)** : Neural empirical Bayes (NEB) 기반 single noise level만으로 훈련 및 샘플링을 단순화함. 이를 통해 복잡성 감소, 안정성 향상.

 Langevin Markov Chain Monte Carlo (Langevin MCMC)를 사용한 Walk 단계와 denoiser를 사용한 Jump 단계를 분리함으로써, 각 단계를 독립적으로 최적화. 이는 모델의 유연성을 높이고, 각 단계에 특화된 기술을 적용

**3. Distributional Conformity Score (DCS)** : 단백질 샘플 품질을 평가하기 위한 새로운 metric 제안. 이를 통해 생성된 샘플이 실제 단백질과 얼마나 유사한지 측정

# Outstanding Paper Award 수상 이유

**Novelty**
- Discrete Walk-Jump Sampling (dWJS) :  단백질 서열과 같은 이산 데이터를 다루기 위해 고안되었으며, 에너지 기반 모델(EBMs)과 스코어 기반 모델을 독창적으로 결합
- Distributional Conformity Score (DCS) : 기존에 부족했던 표준화된 도구를 제공

**Efficiency** 
- single noise level 기반 훈련과 샘플링 과정을 통한 간소화, 계산 부담 감소, 접근성 향상

**Performance** 
- 생성된 샘플의 97-100%가 실험실에서 성공적으로 발현
- 기존 baselines 대비 샘플 품질, 다양성, 계산 효율성 측면에서 능가

**Reply by hoon99** [reply]
### Problem
- Discrete sequence generation은 샘플링이 어렵고, 에너지 기반 모델(EBM)은 훈련과 샘플링이 불안정하며, 디퓨전 모델은 느리고 복잡함.
- 특히 항체 단백질 생성은 가능한 서열 공간이 매우 크고, 실험 가능한 항체 서열은 극히 일부에 불과해 샘플 효율성과 다양성이 중요.
- 기존 방식은 다중 노이즈 스케줄, 높은 계산 자원, 불안정한 샘플 품질 등의 한계를 가짐.

### Outstanding Paper 수상 이유
- Discrete Walk-Jump Sampling(dWJS) 방식은 복잡한 다단계 샘플링 없이도 빠르고 안정적으로 고품질의 단백질 서열을 생성하며, 실제 항체 실험에서도 높은 발현률과 결합률을 기록
- 수학적으로 정교한 샘플링 이론(NEB)을 바탕으로 실험 결과에 있어 다른 SOTA 모델 GPT-4, SeqVDM, IgLM 등)과 비교해 샘플 품질, diversity, 실제 발현률, 결합률에서 우수한 성능 기록

**Reply by jiny419** [reply]
### Outstanding paper 
- SDS는 discrete data에 적합한 새로운 샘플링 패러다임으로, EBM과 score-based model을 분리하여 훈련하는 방식으로 학습을 단순화함.
- dWJS는 효율적인 non-autoregressive 샘플링을 가능하게 하며, 기존의 느린 확산 모델이나 불안정한 EBM 샘플링 문제를 해결.
- 단일 노이즈 레벨만 사용하므로 학습과 샘플링이 훨씬 빠르고 안정적임.

**Reply by kteacher94** [reply]
수상의 이유

- EBM 및 스코어기반 모델의 장점을 결합한 dWJS 샘플링 기법을 통해 기존 단백질 생성 모델의 한계점 개선

- 실험적으로 실제 단백질 생성 모델의 성능을 높은 생성 항체 단백질 발현률 등을 통해 증명한 점

- 논문에서 제시하는 기법이 의약 분야 등에 적용되어 큰 영향을 끼칠 수 있는 점
